### DISCUSSION TITLE
hide thumb image

### DISCUSSION THREAD

**Discussion by ssubham** [gilbarbara/react-spotify-web-playback#72] [Q&A] [answered]
Hi,
I want to hide thumb image what it is being shown on player. 
How can it be possible there as, it has not been provided in setting?

**Comment by gilbarbara** [comment]
It's not possible.

**Reply by ssubham** [reply]
Oh. :( 

Thanks for information.

**Comment by gilbarbara** [comment, marked as answer]
This has been added in v0.13.x with the `hideCoverArt` prop.
